draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1094,28 +1094,28 @@
    caches can use them to keep their contents up to date.
 </t>
 <t>
-   A cache &MUST; invalidate the effective Request URI
-   (<xref target="effective.request.uri"/>) as well as the URI(s) in the
+   A cache &MUST; invalidate the target URI
+   (<xref target="target.resource"/>) as well as the URI(s) in the
```

Comment:
This (existing text) phrase is odd -- shouldn't it say that the cache invalidates the stored responses associated with the target URI, rather than invalidate the URI itself?

Reply:
See below.